lib/uber_task/internal.rb

```diff
@@ -5,8 +5,7 @@ module Internal
     def self.trace
       return unless Thread.current[:__uber_task_trace__]
 
-      message = yield
-      puts message
+      UberTask.logger.debug yield
```

Comment:
Not quite sure that `debug` level should be used - maybe `info` is more appropriate

@borela WDYT?

Reply:
Debug level sounds good, this tracing is only useful when debugging otherwise it will spam a lot as it enter/exits a task.